app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,58 @@
+class RecipeFoodsController < ApplicationController
+  before_action :set_recipe_food, only: %i[edit update destroy]
+  before_action :set_foods, only: %i[edit new update]
+
+  def new
+    @recipe = Recipe.find(params[:recipe_id])
+    @recipe_food = @recipe.recipe_foods.new
+    @recipe_food.save
+  end
+
+  def create
+    @recipe_food = RecipeFood.new(recipe_food_params)
+
+    respond_to do |format|
+      if @recipe_food.save
+        format.html do
+          redirect_to recipe_path(@recipe_food.recipe), notice: 'Recipe food was successfully created.'
+        end
+      else
+        set_foods
+        format.html { render :new, status: :unprocessable_entity }
+      end
+    end
```

Comment:
- [ ] Currently the method create is not creating a new recipe, it is not a core requirement in this project, but since you have a button `create recipe` in your view, kindly make it work or remove the button

Reply:
- Nice work @KayLemba :+1: